app/main.py

```diff
@@ -1,3 +1,58 @@
+from __future__ import annotations
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: (int, float)) -> None:
+        self.km = km
```

Comment:
It would be good to add a check in the constructor to ensure that `km` is a positive number. Distances can't be negative, so this would help prevent misuse of your class.